lgc/patch/llpcPatchPeepholeOpt.cpp

```diff
@@ -229,8 +229,25 @@ void PatchPeepholeOpt::visitBitCast(
         insertAfter(*pBitCastRhs, *pBitCastLhs);
 
         // Create our new shuffle instruction.
+
+        // TODO: Simplify by using pShuffleVector->getShuffleMask() (no arguments) when it becomes available.
+        SmallVector<int, 8> MaskArray;
+        pShuffleVector->getShuffleMask(MaskArray);
+
+        auto pInt32Type = Type::getInt32Ty(pShuffleVector->getContext());
+        SmallVector<Constant*, 8> Elts;
```

Comment:
Elts -> masks

Reply:
This is only a temporary change, but I will make the suggested changes anyway.

Reply:
Thanks for clarification :)